lib/evaluation/warn.ts

```diff
@@ -0,0 +1 @@
+export class Warn extends Error {}
```

Comment:
are you using this class somewhere?

Reply:
I'm using it in EvaluationResults, just had to update it. Error.stack is optional